packages/relay/src/formatters.ts

```diff
@@ -301,6 +301,10 @@ const getFunctionSelector = (data?: string): string => {
   return data.replace(/^0x/, '').substring(0, 8);
 };
 
+const tinybarsToWeibars = (value) => {
+  return value * constants.TINYBAR_TO_WEIBAR_COEF;
```

Comment:
Add some checks for invalid formats.
For example null, or negative numbers, how do we handle those?

Also there's a max value for tinybar we can cap any inputs to that to avoid invalid responses

Reply:
Good point, we can mostly rely on the MN source, but doesn't hurt to guard